ci-operator/jobs/tnozicka/openshift-acme/tnozicka-openshift-acme-postsubmits.yaml

```diff
@@ -0,0 +1,24 @@
+postsubmits:
+  tnozicka/openshift-acme:
+  - agent: kubernetes
+    decorate: true
+    name: branch-ci-tnozicka-openshift-acme-master-images
+    skip_cloning: true
+    spec:
+      containers:
+      - args:
+        - --artifact-dir=$(ARTIFACTS)
+        - --target=[images]
+        - --promote
```

Comment:
Without a specific promotion config this will push to the Origin release -- @smarterclayton do we have sign-off on that

Reply:
I didn't want to promote it intentionally. I have removed the promote section from the ci-config but this got generated.